README.md

```diff
@@ -6,6 +6,7 @@ and multiply by [thirteen][13].
 
 From [Wikipedia][13]:
 > 13 (thirteen /θɜrˈtiːn/) is the natural number following 12 and preceding 14.
+>![alt tag](https://upload.wikimedia.org/wikipedia/commons/f/f3/13_%28ALESA%29.jpg)
```

Comment:
Love the image. Write now it appears in the middle of the wikipedia quote though (as a quote block because of the `>`. Can you move it above the wikipedia entry and remove the quote block formatting?